pages/glossary/visual-section-of-content.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Visual section of content
+key: visual-section-of-content
+unambiguous: true
+objective: false
+---
+
+A _visual section of content_ is a distinct part of the [visible][] content of a document which:
+
+- **uninterrupted** all the content of a visual section of content must be rendered together, without being interrupted by content from another visual section of content, except if it is fully included in the visual section of content (subsection); and
+- **large enough** a visual section of content must contain at least all the content of one [paragraph][], or at least all the content of one [embedded content][] element; and
+- **separated** a visual section of content must be separated from surrounding content by one or more of the following visual cues:
```

Comment:
Good point…
I'm not sure what is the determining characteristic that makes us see that as different sections. We could go further with
```html
<p>1. Foo. Lorem…</p>
<p>Ipsum…</p>
…
<p>2.  Bar. Lorem…</p>
<p>Ispum…</p>
…
```

I am not sure we can really define unambiguously what makes these sections of content…

I guess we have to live with the fact that our "visual sections of content" are not perfect. As long as this results in us having a bit too many inapplicable cases, this is OK.

It would be a problem, for example, if we want to create a rule like "every content is included in a section of content" (but then maybe such a rule wouldn't make much sense for visual sections of content anyway…)

Reply:
I think what makes us see different sections in there is a combination of two things:
1 - A paragraph that is a very short sentence between larger paragraphs. In this context (a book) we recognize that as a section heading unless that very short sentence is recognized as part of a dialogue (starting with - ). This is enough for us.
2 - With the first one being enough, this works to reinforce it. The short sentence starts with a number, distinguishing it from the format of the other paragraphs. If we had multiple sentences like this one after the other we would treat that as a list. Because they are separated we treat them as headings (which are part of a special list - the table of contents).

As I said before, this does not impact the rule, so I'm okay with it.